src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx

```diff
@@ -234,6 +234,7 @@ export function DiscoverGridFlyout({
         onResize={setFlyoutWidth}
         css={{
           maxWidth: `${isXlScreen ? `calc(100vw - ${DEFAULT_WIDTH}px)` : '90vw'} !important`,
+          'z-index': (euiTheme.levels.flyout as number) + 2, // For its usage in the Security Solution timeline, we need this flyout to be above the timeline flyout (which has a z-index of 1002)
```

Comment:
@PhilippeOberti , it will make sense to change the `z-index` only in timeline and not in discover directly.

I guess it has simply be done in css in Security Solution very similar to how it is done here : https://github.com/elastic/kibana/pull/181267/files#diff-60073f16fe1e965e146c61196afa65f33294347228ec86227f02bbbbe848811aR174

Reply:
ooooh that's great, let me update the PR with that change then. At least we can get it in for `8.14` without messing up Discover side of things!

Thanks so much @logeekal

Reply:
@logeekal @michaelolo24 I reverted my original commit, updated the PR description and implemented the logic according to Jatin's recommendation (see [commit](https://github.com/elastic/kibana/pull/182296/commits/a1cf66fc3234422a4356c74e0f06da5cf756eedc))

Reply:
per @logeekal's comment, I just pushed a [small update](https://github.com/elastic/kibana/pull/182296/commits/e8f225d22f9c6efa88e91004515ed73dca7e460d) to guarantee to only target the Discover flyout (the previous code would have impacted all flyouts)